### DISCUSSION TITLE
Issues Getting Spotify Segment To Work

### DISCUSSION THREAD

**Discussion by Zippy-SE** [JanDeDobbeleer/oh-my-posh#1551] [Troubleshoot]
Hello,

I have been checking out some the the posh prompts lately and they are pretty cool. I am having trouble adding the Spotify segment to the existing Paradox theme JSON file. Would I just copy the code from the segment section and paste it into the existing segments within paradox? Is there an order to how the segments go together?  When setting the theme there is no change.  I am using Powershell in Windows 10 to do my work in.  I am stuck on this project to get it to work currently. Thank you.

**Comment by lnu** [comment]
the segments are displayed in order. Should you add the following before the exit segment:
```
        {
          "type": "spotify",
          "style": "powerline",
          "powerline_symbol": "\uE0B0",
          "foreground": "#ffffff",
          "background": "#1BD760",
          "properties": {
            "prefix": " \uE235 "
          }
        }
```
and you should have a working spotify segment(if spotify desktop is installed, not the browser version)
